com.voxdsp.Borg.yaml

```
@@ -0,0 +1,27 @@
+app-id: com.voxdsp.Borg
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: borg
+rename-icon: borg
+rename-desktop-file: borg.desktop
+rename-appdata-file: borg.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: borg
+    buildsystem: simple
+    build-commands:
+      - cc main.c -Ofast -lSDL2 -lm -o borg
+      - install -Dm 0755 borg -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/borg.desktop ${FLATPAK_DEST}/share/applications/borg.desktop
+      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.appdata.xml
```

```suggestion
      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.metainfo.xml
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename